server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -627,13 +690,20 @@ namespace
                 return;
             }
 
-            logger::delay_server::debug("Enqueueing rule ID [{}]", rule_id);
+            // TODO This check may be incorrect. Need to think about it more.
+            // Skip any rules which have been tagged.
+            if (!result[1].empty()) {
+                log_ds::trace("Delay rule was tagged with [{}] earlier. Ignoring rule ID [{}] for now.", result[1], rule_id);
```

Comment:
Handle TODO

Reply:
Removed this check now that the query to retrieve delay rules can exclude claimed rules.